modules/byob/main.tf

```diff
@@ -7,4 +7,6 @@ module "storage" {
   location            = var.location
 
   deletion_protection = var.deletion_protection
+
+  container_name = "wandb"
```

Comment:
nitpick: This might be a bit confusing given that we're dealing with K8s, etc., etc., but I understand this is the Azure nomenclature. Perhaps consider calling this "object container" or "blob container?"

Reply:
oh no that's a really good call out. thank you. we already had a container_name variable for the k8s container and I didn't even realize